templates/ts-react-apollo-client/src/generated-types.tsx

```diff
@@ -15,7 +15,7 @@ export type Scalars = {
 
 /**  @model  */
```

Comment:
I think our config should have enabled subscriptions for crud. Generated file has that disabled and that is why we see changes. Let's change all crud options in graphqlrc file and regenerate it again

Reply:
So `ts-react-apollo-client` does not have a `.graphqlrc.yml`. I used the config in `ts-apollo-postgres-backend`; this one doesn't enable subscriptions. The config in `ts-apollo-mongodb-backend` does enable subscriptions; Is this the config used to generate `ts-react-apollo-client` grapql documents?

Reply:
Yes.. but we should enable all actually. I think they were reverted or something. We should enable all CRUD flags in this PR. Without that we can't really test it properly. Also types here were generated with all crud enabled.